lib/Feature/Feature.cpp

```diff
@@ -78,4 +52,45 @@ std::string NumericFeature::toString() const {
 
 std::string BinaryFeature::toString() const { return Feature::toString(); }
 
+bool Feature::operator<(const vara::feature::Feature &F) const {
```

Comment:
For a `operator<` this is quite complex, so we should test this well, do the test cover all edge cases?

Reply:
I tested all pairwise relations on binary trees with height 2, basic cases and also unrelated features.